wandb/sdk/launch/utils.py

```diff
@@ -430,12 +430,26 @@ def _fetch_git_repo(dst_dir: str, uri: str, version: Optional[str]) -> None:
         except git.exc.GitCommandError as e:
             raise ExecutionError(
                 "Unable to checkout version '%s' of git repo %s"
-                "- please ensure that the version exists in the repo. "
+                "- please ensure that the version exists in the repo. \n"
                 "Error: %s" % (version, uri, e)
             )
     else:
-        repo.create_head("master", origin.refs.master)
-        repo.heads.master.checkout()
+        version = "master"  # change to "main" for current github practice?
```

Comment:
Lets add a master/main fallback. Start with main and fallback to master. That way we prefer the current practice but dont drop support for older repos